dlhub_sdk/client.py

```diff
@@ -112,38 +106,39 @@ def get_task_status(self, task_id):
         r = self.get("{task_id}/status".format(task_id=task_id))
         return r.json()
 
-    def describe_servable(self, servable_id=None, servable_name=None):
+    def describe_servable(self, author, name):
```

Comment:
Why is the function to get_servables called describe_servable? Is this intentional?

Reply:
Seems like the docstring for this command is wrong. The command is to get the description for a specific servable. I've updated accordingly

Reply:
Would it be better to name this method "get_servable_description" anyway?